CHANGES.adoc

```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+== 0.0.35 Jan 01 2020
+
+- Fixes for `CurrentAccess` resource.
+
```

Comment:
Also, please split this to a seperate commit. Its custom in this repo that changes made to `CHANGES.adoc` will be in a commit titled `Release x.x.x`.